include/tcp/socket_reader.h

```diff
@@ -119,47 +91,13 @@ class SocketHandler {
     return socket_fd;
   }
 
-  void sendHeader(int socket_fd, unsigned long long content_length,
-                  std::string content_type) {
-    char header[1024] = {0};
-    sprintf(header, kFormatHeader, content_length, content_type.c_str());
-    writeBuffer(socket_fd, header, strlen(header));
-  }
-
-  void sendFile(int socket_fd, std::string file_path,
```

Comment:
i removed all file handler related stuff for now, will add back as we will add support for sending files within domino::http::Response